tests/reporter/kubernetes.go

```diff
@@ -17,13 +17,23 @@ import (
 	"github.com/onsi/ginkgo/types"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 
 	v12 "kubevirt.io/client-go/api/v1"
 	"kubevirt.io/client-go/kubecli"
 	"kubevirt.io/client-go/log"
 	"kubevirt.io/kubevirt/tests"
 )
 
+const (
+	sriovEntityURITemplate       = "/apis/sriovnetwork.openshift.io/v1/namespaces/%s/%s/"
+	sriovNamespace               = "sriov-network-operator"
```

Comment:
This should be configurable. When we run KubeVirt tests on arbitrary cluster (e.g. OpenShift), the namespace would be different. Passing this as a variable would allow us to run this anywhere.